docs/install_guides/installing-openvino-docker-linux.md

```diff
@@ -277,13 +254,24 @@ docker run --device=/dev/ion:/dev/ion -v /var/tmp:/var/tmp -ti <image_name>
 
 In some cases, the ion driver is not enabled (for example, due to a newer kernel version or iommu incompatibility). `lsmod | grep myd_ion` returns empty output. To resolve, use the following command:
 ```sh
-docker run --rm --net=host -v /var/tmp:/var/tmp –ipc=host  -ti <image_name>
+docker run -it --rm --net=host -v /var/tmp:/var/tmp –ipc=host <image_name>
 ```
 > **NOTE**:
 > - When building docker images, create a user in the docker file that has the same UID and GID as the user which runs hddldaemon on the host.
 > - Run the application in the docker with this user.
 > - Alternatively, you can start hddldaemon with the root user on host, but this approach is not recommended.
 
+### Run demos in the Docker* Image 
+
+If you want to try some demos then run image with the root privileges (some additional 3-rd party dependencies will be installed):
```

Comment:
--> To run the Security Barrier Camera Demo on a specific inference device, run the following commands (additional third-party dependencies will be installed):